pkg/integrations/v4/dm/emitter_no_register.go

```diff
@@ -83,7 +83,7 @@ func (e *nonRegisterEmitter) Send(dto fwrequest.FwRequest) {
 		}
 
 		for _, event := range dataSet.Events {
-			normalizedEvent := legacy.NormalizeEvent(elog, event, labels, integrationUser, dataSet.Entity.Name)
+			normalizedEvent := legacy.NormalizeEvent(elog, event, labels, extraAnnotations, integrationUser, dataSet.Entity.Name)
```

Comment:
I am not able to completely understand the changes you have made.

I debugged the code and follow the same route we followed when we paired. So when I put a breakpoint [here](https://github.com/newrelic/infrastructure-agent/blob/decorate_events_with_custom_attributes/pkg/integrations/v4/emitter/emitter.go#L118) I saw that `dto.LabelsAndExtraAnnotations()` exits with an empty maps for labels and extraAnnotations. So I thought that it would be easier to simply get rid of that distinction and call it a day. 

So when I put another breakpoint here I see this:
![Screenshot 2022-01-27 at 17 26 27](https://user-images.githubusercontent.com/53659978/151402019-314dc56d-7363-4a8a-ae0c-57b31f9dedda.png)
![Screenshot 2022-01-27 at 17 32 22](https://user-images.githubusercontent.com/53659978/151402112-4865083f-6ca7-4666-8c0d-41dcad27abb9.png)

Attributes are already decorated and `labels` and `extraAnnotations` are now useless so this change is.

So now I'm completely lost.

Reply:
The same happens to metrics I have also tested.

Reply:
I don't understand the concern 🤷‍♂️  extraAnnotations where taking into account in metrics but not in events. I have added extraAnnotations to events too.

When you say `get rid of that distinction`. Do you mean the distinction between labels and annotations?
Because these have different meaning. Both are attributes inside events/metrics but labels have `label.` prefix.

Let me know if it is not still clear and maybe it's better to have 5' sync talk ;-)

Reply:
We paired and was not testing things properly because I forgot to add `is_forward_only: true` to this test and it was not testing with THIS use case.

I mark this as resolved because it works as expected.